mne/viz/tests/test_3d.py

```diff
@@ -235,8 +236,25 @@ def test_plot_dipole_locations():
                   subjects_dir, mode='foo')
 
 
+@testing.requires_testing_data
+@requires_version('matplotlib', '1.4')
+@requires_nibabel
+def test_plot_dipole_3d():
+    """Test mpl dipole plotting."""
+    dipoles = read_dipole(dip_fname)
+    trans = read_trans(trans_fname)
+    fig = plot_dipole_3d(dipoles, trans, 'sample', subjects_dir)
+    fig.canvas.scroll_event(0.5, 0.5, 1)  # scroll up
+    fig.canvas.scroll_event(0.5, 0.5, -1)  # scroll down
+    fig.canvas.key_press_event('up')
+    fig.canvas.key_press_event('down')
+    fig.canvas.key_press_event('a')  # some other key
+    matplotlib.pyplot.close('all')
```

Comment:
This is unsafe. Doing `import matplotlib` doesn't automatically import the `matplotlib.pyplot` namespace. This will only work if we have by chance done `import matplotilb.pyplot` elsewhere, so you should explicitly import it here (as we always do) to be safe